src/assets/wise5/classroomMonitor/dataExport/strategies/LabelComponentDataExportStrategy.ts

```diff
@@ -0,0 +1,77 @@
+import { Component } from '../../../common/Component';
+import { ComponentDataExportParams } from '../ComponentDataExportParams';
+import { ComponentRevisionCounter } from '../ComponentRevisionCounter';
+import { AbstractComponentDataExportStrategy } from './AbstractComponentDataExportStrategy';
+
+export class LabelComponentDataExportStrategy extends AbstractComponentDataExportStrategy {
+  maxNumLabels: number;
+
+  constructor(component: Component, additionalParams: ComponentDataExportParams) {
+    super(component, additionalParams);
+    this.maxNumLabels = 0;
+  }
+
+  export(): void {
+    this.controller.showDownloadingExportMessage();
+    const components = [{ nodeId: this.component.nodeId, componentId: this.component.id }];
+    this.dataExportService.retrieveStudentDataExport(components).then((result) => {
+      this.populateColumnNames(this.columnNames, this.columnNameToNumber);
```

Comment:
Does this ```populateColumnNames()``` call need to be here (after ```retrieveStudentDataExport()```), or can it be called sooner, inside the constructor of ```AbstractComponentDataExportStrategy```? Does it need to pass in the two params to the function, or can the func get it from the class fields?